ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/423__model_serving_customruntimes.robot

```diff
@@ -81,5 +110,38 @@ Create Test Serving Runtime Template If Not Exists
         Log    message=Creating the necessary Serving Runtime as part of Test Setup.
         Open Dashboard Settings    settings_page=Serving runtimes
         Upload Serving Runtime Template    runtime_filepath=${OVMS_RUNTIME_FILEPATH}
-        Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}        
+        ...    serving_platform=multi
+        Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}
+        ...    serving_platform=multi
     END
+
+Generate Runtime YAMLs
+    [Documentation]    Generates three different Custom Serving Runtime YAML files
+    ...                starting from OVMS one. Each YAML will be used for a different
+    ...                supported serving platform (single model, multi model, both)
+    Set Suite Variable    ${RUNTIME_BOTH_FILEPATH}    ${RESOURCES_DIRPATH}/csr_both_model.yaml
+    Set Suite Variable    ${RUNTIME_SINGLE_FILEPATH}    ${RESOURCES_DIRPATH}/csr_single_model.yaml
+    Set Suite Variable    ${RUNTIME_MULTI_FILEPATH}    ${RESOURCES_DIRPATH}/csr_multi_model.yaml
```

Comment:
These three `${RESOURCES_DIRPATH}/csr_*` are used on two places. Maybe it would make sense to create a global variables in this file for them?

Reply:
these will be used only by one test, hence I made these suite variable only when running that particular test